2-mandatory/2-oxygen-levels.js

```diff
@@ -11,7 +11,15 @@
     Some string methods that might help you here are .replace() and .substring().
 */
 
-function findSafeOxygenLevel() {}
+function findSafeOxygenLevel(oxygenLevels) {
+  let oxygenLevel = oxygenLevels
+    .filter((e) => /^\d/.test(e))
+    .filter((e) => /%$/g.test(e))
+    .map((e) => e.replace("%", ""))
+    .find((n) => n > 19.5 && n < 23.5)
```

Comment:
I really like this solution because it breaks down the problem into clear steps and uses the `Array.map` and `Array.filter` functions to clearly indicate what the objective is for the code. What is the type of the variable `n` here?

Reply:
I didn't convert string to number ...